addons/website/models/website.py

```diff
@@ -1437,6 +1437,9 @@ def _is_snippet_used(self, snippet_module, snippet_id, asset_version, asset_type
             match = re.search('<([^>]*class="[^>]*)>', snippet_template_html)
             snippet_occurences.append(match.group())
 
+        if self._check_snippet_used(snippet_occurences, asset_type, asset_version):
+            return True
```

Comment:
Indeed, it seems like a good thing to do. Although, if I am not mistaken, you are re-calling `_check_snippet_used` with the already checked "snippet occurrences" the second time you call it. Maybe you can improve this? :+1:

Reply:
done